examples/js/loaders/sea3d/SEA3D.js

```diff
@@ -4749,7 +4159,7 @@ SEA3D.EventDispatcher.prototype = {
 
 };
 
-SEA3D.EventDispatcher.apply = function ( object ) {
+SEA3D.EventDispatcher.apply = function( object ) {
 
 	object.addEventListener = SEA3D.EventDispatcher.prototype.addEvenListener;
```

Comment:
Not that this line has been changed, but it jumped right at me skimming the code: Should it read `addEventListener` - note the extra "t"?

Also worth mentioning: We have recently deprecated `THREE.EventListener.prototype.apply` and instead do
```javascript
Object.assign( AClass.prototype, THREE.EventListener.prototype /* , { ... methods } */ );
```
now (there's a polyfill in `Three.js`, so yes, this works with all browsers).